src/test/java/seedu/address/person/logic/commands/AddCommandTest.java

```diff
@@ -147,6 +154,11 @@ public void setPerson(Person target, Person editedPerson) {
         public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public Person getPersonByName(String name) {
+            throw new AssertionError("This method should not be called.");
+        }
     }
 
     /**
```

Comment:
ERROR:src/test/java/seedu/address/transaction/commands/AddCommandTest.java:11: no newline at EOF.

You wanna fix this so we won't fail Travis? Tq!

Reply:
I realised that the multimap and tokenizer in Overview tab is not integrated. Is it supposed to be like this?